coopertunes/datasets/AudioDataset.py

```diff
@@ -0,0 +1,69 @@
+import torch
+import torch.utils.data
+import torch.nn.functional as F
+
+from librosa.core import load
+from librosa.util import normalize
+
+from pathlib import Path
+import numpy as np
+import random
+
+
+def files_to_list(filename):
+    """
+    Takes a text file of filenames and makes a list of filenames
+    """
+    with open(filename, encoding="utf-8") as f:
+        files = f.readlines()
+
+    files = [f.rstrip() for f in files]
+    return files
+
+
+class AudioDataset(torch.utils.data.Dataset):
+    """
+    This is the main class that returns audio frames
+    """
+
+    def __init__(self, training_files, segment_length, sampling_rate, augment=True):
```

Comment:
please move it to hparams

Reply:
Already in hparams, imo at this level it should be steered by top level abstraction (here model supervisors with its own hparams)